src/users/user.testData.ts

```diff
@@ -68,3 +68,15 @@ export const userArray = [
     otpExpiresAt: null,
   },
 ];
+
+export const unconnectedAccount = {
```

Comment:
Maybe you should add more test cases such as googleId  is empty string, or googleId is numerical.

Reply:
> Maybe you should add more test cases such as googleId is empty string, or googleId is numerical.

Thanks for reminding me of that! However, after going through other tests, I haven't found any of them checked the type of parameters. Maybe that's because Nest will help us do that based on decorators in "dto" files? Based on this thought, I updated the decorators of the ConnectAccountDto and hoped this could be another way to ensure the integrity of incoming parameters of requests.